pkg/js/helpers.js

```diff
@@ -274,15 +262,17 @@ function format_tt(transform_table) {
 }
 
 // IMPORT_TRANSFORM(translation_table, domain)
-function IMPORT_TRANSFORM(translation_table, domain,ttl) {
-    return function(d) {
-        var rec = addRecord(d, "IMPORT_TRANSFORM", "@", domain, [
-            {'transform_table': format_tt(translation_table)}])
-        if (ttl){
-            rec.ttl = ttl;
-        }
-    }
-}
+var IMPORT_TRANSFORM = recordBuilder('IMPORT_TRANSFORM', {
```

Comment:
In this code the TTL parameter to IMPORT_TRANSFORM() is removed. It needs to be included. The syntax shouldn't change.